content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: 'Prisma Client extensions'
+metaTitle: 'Prisma Client extensions (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+preview: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+This page is in draft.
+
+</Admonition>
+
+Use Prisma Client extensions to add functionality to your models, result objects, queries, or to Prisma Client itself.
+
+There are four types of extension. You can use each type to customize a different aspect of Prisma Client.
```

Comment:
```suggestion
There are four types of extensions, also called extension layers, which you can use to customize different aspects of Prisma Client.
```

Reply:
If "extension layers" is a terminology we want to "officially" introduce in the context of Prisma Client extensions, I think we should make this an explicit concept and mention it in a heading somewhere too. (If we don't want to use this as "official" terminology, I'd rather omit a mention of it to keep things simple).

Reply:
Yes, I think we should pick one term and settle on that.